package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "gherlint",
-    "version": "1.0.0-alpha.1",
+    "version": "0.0.1",
```

Comment:
why are we downgrading the version?

Reply:
we never were at 1.0.0. that was my mind playing around.